tests/conftest.py

```diff
@@ -0,0 +1,28 @@
+import sys
+sys.path.insert(0, 'D:\Luther\Self-Learning\Chat App\VL_Chat_App')
```

Comment:
You cannot add a local path from your computer to the codebase like this...

Reply:
Same as above, I haven't finished fixing the test yet. This commit is just a demo.